### PR TITLE
Aws polly engine fix

### PR DESCRIPTION
**Pull request by vdharashive**
engine parameter was  not able to change using config synthesizer

### CODE REVIEW

**Review by xquanluu — APPROVED** [review]
look good to me

File: lib/tasks/say.js [hunk 1]
```diff
@@ -66,7 +66,8 @@ class TaskSay extends TtsTask {
     const {updateSpeechCredentialLastUsed} = require('../utils/db-utils')(this.logger, srf);
     const {writeAlerts, AlertType, stats} = srf.locals;
     const {synthAudio} = srf.locals.dbHelpers;
-    const engine = this.synthesizer.engine || 'standard';
+    const engine = this.synthesizer.engine || cs.synthesizer.engine || 'standard';
```
Comment (davehorton) [inline lib/tasks/say.js:69]: should be
```
 const engine = this.synthesizer.engine || cs.?synthesizer.engine || 'standard';
```

**Review by davehorton — CHANGES_REQUESTED** [review]
you should make a similar change in tts-task.js

File: lib/tasks/say.js (hunk 1 above)
Comment (vdharashive) [reply lib/tasks/say.js]: @davehorton have implimented the suggested changes also changed from standard to neural , since it much batter

**Review by davehorton — APPROVED** [review]

File: lib/tasks/say.js [hunk 2]
```diff
@@ -66,7 +66,8 @@ class TaskSay extends TtsTask {
     const {updateSpeechCredentialLastUsed} = require('../utils/db-utils')(this.logger, srf);
     const {writeAlerts, AlertType, stats} = srf.locals;
     const {synthAudio} = srf.locals.dbHelpers;
-    const engine = this.synthesizer.engine || 'standard';
+    const engine = this.synthesizer.engine || cs.synthesizer?.engine || 'neural';
+
```
Comment (davehorton) [inline lib/tasks/say.js:70]: fix lint issue (trailing space).  Please run
```
npm run jslint
```
before making PRs

**Review by davehorton — APPROVED** [review]

**Review by davehorton — APPROVED** [review]

**Pull request merged**